src/main/java/org/sopt/confeti/api/user/facade/UserOnboardFacade.java

```diff
@@ -81,6 +97,23 @@ public UserOnboardTopArtistsDTO getTopArtists(int limit) {
 
     public UserOnboardTopArtistsDTO getTopArtists(int limit, long userId) {
         List<ConfetiArtist> topArtists = getAllTopArtists();
+        UserOnboardCacheDTO cachedArtists = userOnboardService.getCachedArtists(userId);
+
+        UserOnboardTopArtistsDTO userOnboardTopArtistsDTO = UserOnboardTopArtistsDTO.from(
+            topArtists.stream()
+                .filter(artist -> !cachedArtists.favoriteArtistIds().contains(artist.getId()))
+                .limit(limit)
+                .toList()
```

Comment:
이 부분 반복되는 것 같은데, 함수화하는건 어떨까요?
```
private List<ConfetiArtist> getFilteredArtists(List<ConfetiArtist> artists, List<String> excludeArtistIds, int limit) {
    return artists.stream()
        .filter(artist -> !excludeArtists.contains(artist.getId()))
        .limit(limit)
        .toList();
}
```

```suggestion
return UserOnboardTopArtistsDTO.from(
    getFilteredArtists(topArtists, cachedArtists.favoriteArtistIds(), limit)
);
```

Reply:
좋은데요?? 바로 반영하겠습니다 ~ 👍